README.md

```diff
@@ -1,14 +1,60 @@
-# Package Modules Working Group
+# [Package Modules Working Group][wg-github]
 
-This working group is there to investigate the possibility of using something like the [module system](https://nixos.org/manual/nixos/stable/index.html#sec-writing-modules) for Nixpkgs packages. See the [discourse post](https://discourse.nixos.org/t/working-group-member-search-module-system-for-packages/26574) for more info.
+This working group is there to investigate the possibility of using something
+like the [NixOS module system][module-system] for Nixpkgs packages. See the
+[discourse post][discourse-thread] for more info.
 
-All working group members have access to this repository. The end goal is to have an RFC developed that can be proposed. Using issues and pull requests to achieve that is encouraged.
+All working group members have access to [this repository][wg-github]. The
+end goal is to have an [RFC](./rfc.md) developed that can be proposed. Using
+[issues][wg-issues] and [pull requests][wg-pulls] to achieve that is encouraged.
+
+## Meetings
+
+We meet **weekly, every Friday at 13:00 UTC** in [this Jitsi][wg-jitsi].
+
+Notes are written live in [this pad][wg-pad] during meetings, and collected
+below afterwards. Presentations are streamed via YouTube, with recordings available on
+[Youtube: @nixpkgs-architecture][yt-nixpkgs-architecture].
+
+### Past Meetings
+
+Notes and/or recordings of past meetings, in reverse chronological order: 
+
+- [2023-06-16: drv-parts presentation (YouTube)][yt-drv-parts]
+- [2023-06-09](./meetings/2023-06-09.md)
+- [2023-06-02](./meetings/2023-06-02.md)
+- [2023-05-26](./meetings/2023-05-26.md)
+- [2023-05-19](./meetings/2023-05-19.md)
+- [2023-05-12](./meetings/2023-05-12.md)
+- [2023-05-05](./meetings/2023-05-05.md)
+- [2023-04-28](./meetings/2023-04-28.md)
```

Comment:
```suggestion
Notes are written live in [this pad][wg-pad] during meetings, and committed to the [./meetings](./meetings) folder afterwards.
Occasional presentations are streamed and recorded to [this YouTube playlist](https://www.youtube.com/playlist?list=PLHG2N-mfvWT48ZGoUC5W6OMMdln0IsNrq) on the Nixpkgs Architecture Team channel.
```

Let's avoid having to update the meeting notes in multiple places.

I also created a playlist for such presentations.